imap/smtpclient.h

```diff
@@ -108,6 +112,11 @@ extern int smtpclient_open_host(const char *addr, smtpclient_t **smp);
 extern int smtpclient_send(smtpclient_t *sm, smtp_envelope_t *env, struct buf *data);
 extern int smtpclient_sendprot(smtpclient_t *sm, smtp_envelope_t *env, struct protstream *data);
 
+/* Perform SMTP envelope (and optionally size and/or From: header address checks)
```

Comment:
This is find hard to parse. Would "Check the SMTP envelope" work instead of "Perform .."